src/System.Configuration.ConfigurationManager/src/System.Configuration.ConfigurationManager.csproj

```diff
@@ -47,6 +47,8 @@
     <Compile Include="System\Configuration\ConfigurationLockCollectionType.cs" />
     <Compile Include="System\Configuration\ConfigurationManager.cs" />
     <Compile Include="System\Configuration\ConfigurationManagerInternalFactory.cs" />
+    <Compile Include="System\Configuration\ConfigurationPermission.cs" />
```

Comment:
Oops, I didn't notice this before. Can these types go into C:\git\corefx\src\System.Security.Permissions\src\System\Configuration ? All the CAS types are there.